sharepoint/sharepoint-ps/sharepoint-online/Set-SPOContainerType.md

```diff
@@ -0,0 +1,175 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/get-spocontainer
+applicable: SharePoint Online
+title: Set-SPOContainerType
+schema: 2.0.0
+author: ShreyasSar26
+ms.author: ShreyasSar26
+ms.reviewer:
+---
+ 
+# Set-SPOContainerType
+ 
+## SYNOPSIS
+ 
+Sets or updates one or more property values of a trial or standard container type.
+ 
+## SYNTAX
+ 
+### ParamSet1
+ 
+```powershell
+Set-SPOContainerType -ContainerTypeId <ContainerTypeId> -OwningApplicationId <OwningApplicationId> -ContainerTypeName <ContainerTypeName>
+```
+### ParamSet2
+```powershell
+Set-SPOContainerType -ContainerTypeId <ContainerTypeId> -AzureSubscriptionId <AzureSubscriptionId> -ResourceGroup <ResourceGroup>
+```
+ 
+## DESCRIPTION
+The Set-SPOContainertype cmdlet is used to reset the parameters associated with a container type - both trial and standard. The cmdlet can be used to change the container type name or the associated billing profile of the contianer type.
```

Comment:
>The Set-SPOContainertype [](http://example.com/codeflow?start=0&length=24)

```suggestion
This
```

Reply:
Is the recommendation to remove the description?